CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ mark_as_advanced(VERSION_MAJOR VERSION_MINOR VERSION_PATCH VERSION_STRING)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/cmake/modules")
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
You also probably want `CMAKE_CXX_STANDARD_REQUIRED` set to true (cf. https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD_REQUIRED.html#prop_tgt:CXX_STANDARD_REQUIRED).